php-frontend/src/main/java/org/sonar/plugins/php/api/tree/declaration/FunctionDeclarationTree.java

```diff
@@ -26,8 +26,15 @@
 
 import javax.annotation.Nullable;
 
+/**
+ * <p><a href="http://php.net/manual/en/functions.user-defined.php">Function Declaration</a>
+ * <pre>
+ *  function {@link #name()} {@link #parameters()} {@link #body()}
+ *  function & {@link #name()} {@link #parameters()} {@link #body()}
+ * </pre>
+ */
 @Beta
-public interface FunctionDeclarationTree extends FunctionTree {
+public interface FunctionDeclarationTree extends FunctionTree, DeclarationTree {
 
   SyntaxToken functionToken();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)